src/main/kotlin/rmc/route/vehicle/VehicleRoutes.kt

```diff
@@ -13,17 +13,57 @@ import rmc.dto.vehicle.CreateVehicleDTO
 fun Route.vehicleRoutes() {
     val vehicleRepository = VehicleRepositoryImpl()
 
-    authenticate {
+//    authenticate {
         route("/vehicle") {
             post("/createVehicle") {
                 val createVehicleDTO = call.receive<CreateVehicleDTO>()
-                val principal = call.principal<JWTPrincipal>() ?: throw AuthenticationFailed()
-                val userId = principal.payload.getClaim("userId")?.asInt() ?: throw AuthenticationFailed()
-
+//                val principal = call.principal<JWTPrincipal>() ?: throw AuthenticationFailed()
+//                val userId = principal.payload.getClaim("userId")?.asInt() ?: throw AuthenticationFailed()
+                val userId = 1
```

Comment:
Dit is een shortcut om te testen neem ik aan?

Reply:
klopt